pkg/ui/workspaces/db-console/src/views/hotRanges/hotRangesTable.tsx

```diff
@@ -44,6 +44,19 @@ interface HotRangesTableProps {
   onSortChange?: (ss: SortSetting) => void;
 }
 
+const tableToCell = (val: any) => {
+  // make an assumption if we see a single table
```

Comment:
nit: clean up comment

Reply:
removed this function since the link isn't quite possible right now.